examples/create_account.rs

```diff
@@ -228,7 +230,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                     }
                     FinalExecutionStatus::SuccessValue(ref s) => {
                         // outcome.status != SuccessValue(`false`)
-                        if s == b"false" {
+                        if s.is_empty() {
```

Comment:
I am surprised by both previous `false` check and `is_empty` check now.

`on_account_create` finishes with a boolean output `true` if account was successfully created: https://github.com/near/near-linkdrop/blob/49279e529c254fa7736465b4a39d05cb8f1e5443/src/lib.rs#L211

Reply:
Kinda weird, but I have run it manually and it returned me ""

Reply:
Maybe it because I have tried to create from myself (sub account)

Reply:
Confirmed. It return nothing if you are creating subaccount, and returns true if you are creating from linkdrop